File: Assignment_1/Assignment1_10.py
"""-----------------------------------------------------------------------------------------
                            Assignment1_10
                    (Student name - Vaishali Jorwekar)
                     Python by Marvellous Infosystems
-----------------------------------------------------------------------------------------
Problem statement :
            This program accepts NAME from user and displays it's length            
------------------------------------------------------------------------------------------"""
#-------------------------------------------------------------------
# This function finds the length of the "name" entered by user
#-------------------------------------------------------------------
def FindLength(name):
    name_length = len(name)
    print(f"Length of name '{name} ' is : ",name_length)   
#-----------------------------------------------------------------
#-------------------------------------------------------------------
# This function accepts a "name" from user and 
# calls FindLength(name)
#-------------------------------------------------------------------
def main():
    print("---------------------------------------------------")
    print("Enter the name : ",end="")
    name = input()          # Accept input
    FindLength(name)   
#------------------------------------------------------------------  
# Entry point of program
#------------------------------------------------------------------
if __name__=="__main__":
    main()
#-----------------------------------------------------------

File: JavaPractice/Assignment1_10.java
/*-----------------------------------------------------------------------------------------
                            Assignment1_10
                    (Student name - Vaishali Jorwekar)
                     Python by Marvellous Infosystems
-----------------------------------------------------------------------------------------
Problem statement :
            This program accepts NAME from user and displays it's length            
---------------------------------------------------------------------------------*/
import java.util.Scanner;
public class Assignment1_10 {
    public static void main(String[] args) {
        Scanner scanObj = new Scanner(System.in);
        System.out.println("Enter the name:");
        String name = scanObj.nextLine();
        int nameLen = name.length();
        System.out.println("Length of '"+name+"' is :"+nameLen);
        scanObj.close();
    }
    
}
